prisma/schema.prisma

```diff
@@ -11,13 +11,15 @@ generator client {
 }
 
 model Company {
-  companyId        Int     @default(autoincrement()) @id
+  id        Int   @default(autoincrement()) @id
   name     String
+  users User[]
 }
 
 model User {
   userId        Int       @default(autoincrement()) @id
-  companyId     Int
+  companyId     Int? 
+  worksAt   Company? @relation(fields: [companyId], references: [id])
```

Comment:
that's just a personal preference, but I prefer having the same name for references as the table name. so in this case I'd name it `company` instead of `worksAt`. this makes it easier to remember when you work with the user model

Reply:
done